EnglishFilter/sections/antiadblock.txt

```diff
@@ -871,7 +871,7 @@ tvchoicemagazine.co.uk#%#//scriptlet("set-constant", "showModal", "noopFunc")
 @@||ams.cdn.arkadiumhosted.com/advertisement/display/stable/display-ads.js$domain=games.metro.us
 games.metro.us#%#//scriptlet("adjust-setInterval", "game will", "", "0.02")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/78293
-freshi.site,cookuve.com,srek.net,exe.app#%#//scriptlet("set-constant", "disableItToContinue", "noopFunc")
+eririo.club,freshi.site,cookuve.com,srek.net,exe.app#%#//scriptlet("set-constant", "disableItToContinue", "noopFunc")
```

Comment:
`@@||googletagmanager.com/gtag/js$domain=eririo.club` for cb platforms.
`freshi.site,cookuve.com,srek.net,exe.app` they use the same script, so add these domains too.